index.js

```diff
@@ -1,3 +1,4 @@
+import 'react-native-get-random-values';
```

Comment:
Porque esse pacote foi instalado?

Reply:
Sem ele obtive o problema listado em: https://github.com/uuidjs/uuid#getrandomvalues-not-supported

Reply:
Ok, ciente